openhtf/plugs/user_input.py

```diff
@@ -166,9 +171,8 @@ def prompt(self, message, text_input=False, timeout_s=None):
       console_prompt.start()
       self._cond.wait(timeout_s)
```

Comment:
If you could split this function here into these 3, it would close another issue out:

* start_prompt(message, text_input) -> Everything above this wait
* wait_for_prompt(timeout_s) -> this wait and down
* prompt(message, text_input, timeout_s) -> just a call to start and wait